### DISCUSSION TITLE
Will a .net 5 project be able to use AOT feature?

### DISCUSSION THREAD

**Discussion by dkboon01** [dotnet/runtime#60158] [Q&A]
I'm really not for sure where to post this question.  Will the AOT feature be available for projects that are built with .net 5 or do you have to use .net 6 to get the AOT feature?  If i'm not in the correct area, then can you suggest where to put this question.

**Comment by ghost** [comment]
I couldn't figure out the best area label to add to this issue. If you have write-permissions please help me learn by adding exactly one [area label](https://github.com/dotnet/runtime/blob/master/docs/area-owners.md).

**Comment by jkotas** [comment]
What is the AOT feature that you are looking for?

**Comment by dkboon01** [comment]
Ahead of time compile.  I was wondering if I had a .net 5 program if I was
able to use AOT to publish.

**Comment by jkotas** [comment]
.NET 5 support ReadyToRun AOT. Here are the docs for how to use it: https://docs.microsoft.com/en-us/dotnet/core/deploying/ready-to-run
